grpc/supervisor.proto

```diff
@@ -1,17 +1,22 @@
 syntax = "proto3";
 
-import "google/protobuf/empty.proto";
-
 package kagami_supervisor;
 
 service Supervisor {
-    rpc worker_report_in (WorkerReportInRequest) returns (google.protobuf.Empty);
-    rpc update_provider_status (UpdateProviderRequest) returns (google.protobuf.Empty);
+    rpc worker_report_in (WorkerReportInRequest) returns (WorkerReportInRequest);
+    rpc update_provider_status (UpdateProviderRequest) returns (UpdateProviderStatusResponse);
+}
+
+message WorkerReportInResponse {
+    string supervisor_addr = 1;
+}
+
+message UpdateProviderStatusResponse {
+    string provider_id = 1;
```

Comment:
type is supposed to be int

Reply:
https://github.com/ProjectSoftoid/Kagami/pull/27/commits/1519436ccab0d32b37d797b13c29a1d35eab13ca